Tests/Tests/PayWithLinkViewController-WalletViewModelTests.swift

```diff
@@ -100,8 +100,12 @@ class PayWithLinkViewController_WalletViewModelTests: XCTestCase {
 extension PayWithLinkViewController_WalletViewModelTests {
 
     func makeSUT() throws -> PayWithLinkViewController.WalletViewModel {
+        // No link_settings in the test JSON, so inject it here
+        // TODO(bmelts): Should this be in the test JSON?
```

Comment:
@eurias-stripe or @davidme-stripe, would love your thoughts on this - I dislike this kind of hack generally but it's not clear to me that an undocumented property belongs in the test JSON.

Reply:
I prefer adding it to the json file, by adding it here you are somewhat breaking the intent of the tests of loading a response as if it came from the server.
Even if the property is undocumented it would be part of the json response, so it belongs in the test json file.
If this only applies to this set of tests you should probably create a separate test json though.

Reply:
Makes sense to me. I'm going to add it to the existing files because a) as you said, it's part of the expected API response, and it's very possible there will be (or should be, at least!) more tests reliant on the response behaving the way it does in practice.